tests/run-tests.sh

```diff
@@ -153,5 +157,5 @@ check_config_files
 setup_workdir
 setup_dss_binary
 check_block_devs "$DEVS"
-run_tests
+run_tests $@
```

Comment:
Since we are opening a separate PR #196 for this, I think we should remove this commit from this PR.